canvas_modules/common-canvas/src/object-model/api-pipeline.js

```diff
@@ -320,61 +320,72 @@ export default class APIPipeline {
 		return node;
 	}
 
-	// Returns a source node for auto completion or null if no source node can be
-	// detected. The source node is either:
+	// Returns a node for auto completion or null if no node can be
+	// detected. The node is either:
 	// 1. The selected node, if only *one* node is currently selected or
 	// 2. The most recently added node, provided it has one or more output ports or
 	// 3. The most-recent-but-one added node, provided it has one or more output ports
-	getAutoSourceNode(autoLinkOnlyFromSelNodes) {
-		var sourceNode = null;
+	//    or if autoLinkToBindingNodes is true, then it may be a binding node
+	getAutoLinkedNode(autoLinkOnlyFromSelNodes, autoLinkToBindingNodes) {
+		var autoLinkedNode = null;
```

Comment:
Instead if using `autoLinkedNode` why not use the name the caller calls it when it is returned which is `nodeToLinkWith`. That way, we are using the same name for the same concept.

Reply:
Good point, updated